editor/animation_track_editor_plugins.h

```diff
@@ -114,7 +114,7 @@ class AnimationTrackEditTypeAudio : public AnimationTrackEdit {
 
 	void _preview_changed(ObjectID p_which);
 
-	bool len_resizing;
+	bool len_resizing = false;
 	bool len_resizing_start;
```

Comment:
Missed this one.

Reply:
For now I haven't touched ones that aren't already initialized in the constructor. But I could go and ensure all of the bools that currently aren't initialized are initialized to false.

Reply:
Let'S do it as a follow up I guess, this one is good as is.